echopype/convert/set_groups_ek60.py

```diff
@@ -6,18 +6,19 @@
 import xarray as xr
 from _echopype_version import version as ECHOPYPE_VERSION
 
-from ..echodata.convention.attrs import (
-    DEFAULT_BEAM_COORD_ATTRS,
-    DEFAULT_PLATFORM_COORD_ATTRS,
-    DEFAULT_PLATFORM_VAR_ATTRS,
-)
+from ..echodata.convention import sonarnetcdf_1
 from ..utils.coding import set_encodings
 
 # fmt: off
 from .set_groups_base import DEFAULT_CHUNK_SIZE, SetGroupsBase
 
 # fmt: on
 
+_varattrs = sonarnetcdf_1.conv.yaml_dict["variable_and_varattributes"]
```

Comment:
Do you see this `sonarnetcdf_1` to have more than the `conv` object? It seems a little awkward to call `.conv` after already getting the `sonarnetcdf_1`. I would expect more of a `yaml_dict` call on that to get to the convention yaml right away.

Reply:
> Do you see this sonarnetcdf_1 to have more than the conv object?

Not at this time. I agree, it's awkward to have `sonarnetcdf_1.conv.yaml_dict`. I ran out of patience trying to figure out how to try to accomplish that via the `__init__.py` after many failed attempts. Can you suggest a solution?